### DISCUSSION TITLE
翻译请求能否迁移到服务端？

### DISCUSSION THREAD

**Discussion by Triabin** [metatube-community/jellyfin-plugin-metatube#302] [Ideas]
家里网络规划是这样的，需要给网络附魔的服务专门使用了一个Linux用户去启动（环境变量），导致了我用该用户启动embyserver的时候，有些权限问题解决不了，难顶

**Comment by xjasonlyu** [comment]
这本来就是在服务端翻译的啊🤔

**Reply by Triabin** [reply]
这里的服务端指的是metatube的服务端，不是emby的服务端哦。我试了，emby在墙内，metatube服务端在墙外，能刮削，但是不能翻译。不过也理解作者这么做的用意，估计是之前提供了几个公益服，然后不想让别人的api-key经过服务端过一遍徒增风险

**Reply by xjasonlyu** [reply]
但翻译确实是在mt服务端做的

**Reply by Triabin** [reply]
好的好的，我再查查日志看看啥情况😭
